.github/render-action/generate.sh

```diff
@@ -46,6 +46,7 @@ function convert_markdown_to_asciidoc {
       -a toclevels=4 \
       --auto-ids \
       --auto-id-prefix="${md_filename%.md}_" \
+      --auto-id-separator="_" \
```

Comment:
This change has to do with fixing links within the document.
You will see a few cases where I changed the links in files not related to the Adventuring section; this is part of why I did that.